drivers/gpu/drm/i2c/adv7511.c

```diff
@@ -581,6 +591,29 @@ static int adv7511_irq_process(struct adv7511 *adv7511)
 			wake_up_all(&adv7511->wq);
 	}
 
+	/*
+	 * reset dsi TX/RX when transfer error
+	 */
+	ret = regmap_read(adv7511->regmap_cec, 0x48, &state);
+	if (ret < 0)
+		return ret;
+	if (powered && (!irq1 || state & BIT(2))) {
```

Comment:
That's because the irq of adv7533 is not reliable when the HDMI cable is plugged in.  If you test it ,without the HDMI cable plugged in , when a dsi RX error occur during booting, you will always get state = 0x04. 
So, ideally, the condition should be ' if (powered && (state & BIT(2)) '. That's the condition that dsi RX error is detected when power on. 

But, unfortunately, for unknown reason, with the HDMI cable plugged in, when the dsi RX error occur during booting, state will not always be 0x04 and it could affect the value of irq0 and irq1, Although it can always trigger a irq, when dsi RX error occur. But I can not judge it by the ideal condition. That's why I also mask the non-dsi interrupts:
+	/* mask non-dsi interrupts */
+	regmap_write(adv7511->regmap, 0x94, 0);

I'm not sure things happened like this on your side.